### ISSUE TITLE
Diamond Princess is not a Canadian province - lines 233 and 240 in time_series_covid19_confirmed_global.csv

### ISSUE THREAD

**Issue by rowrowrowyourboat** [CSSEGISandData/COVID-19#1562]
Lines 233 and line 240 in 'time_series_covid19_confirmed_global.csv' file show Canada for the country but the State/province as 'Diamond Princess' and 'Recovered'.

**Comment by mainadwitiya** [comment]
Yeah and if you try to make a chart of recovered cases dynamically country wise, It will show Index 0 is out of bound at axis=0
